### PR TITLE
Zib-HelpFromOthers

### PR DESCRIPTION
**Pull request by Ashwin-nictiz**

### CODE REVIEW

File: resources/zib/zib-HelpFromOthers.xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Organization" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/CareTeam" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/HealthcareService" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-Practitioner" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactPerson" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthcareProvider-Organization" />
```
Comment (LuudSlagter) [inline resources/zib/zib-HelpFromOthers.xml:114]: It makes more sense to refer to zib-HealthcareProvider (profile on Location) here, as it is the focal profile of the zib HealthcareProvider.

File: resources/nl-core/nl-core-HelpFromOthers.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="CarePlan" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-HelpFromOthers" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="CarePlan">
+      <path value="CarePlan" />
+      <alias value="nl-core-HelpFromOthers" />
+    </element>
```
Comment (LuudSlagter) [inline resources/nl-core/nl-core-HelpFromOthers.xml:30]: Add reference to nl-core-Patient on `.subject`.

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

**Comment by LuudSlagter** [comment]
There are still some validation errors that need to be resolved.

File: resources/nl-core/nl-core-HelpFromOthers.xml (hunk 2 above)
Comment (Ashwin-nictiz) [reply resources/nl-core/nl-core-HelpFromOthers.xml:30]: Fix

File: resources/zib/zib-HelpFromOthers.xml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      <mapping>
+        <identity value="zib-helpfromothers-v3.01-2020EN" />
+        <map value="NL-CM:3.2.1" />
+        <comment value="HelpFromOthers" />
+      </mapping>
+    </element>
+    <element id="CarePlan.category">
+      <path value="CarePlan.category" />
+      <min value="1" />
+      <max value="1" />
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-HelpFromOthers.xml:45]: Fix

File: resources/zib/zib-HelpFromOthers.xml [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="243114000" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-HelpFromOthers.xml:52]: Fix

File: resources/zib/zib-HelpFromOthers.xml [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    <element id="CarePlan">
+      <path value="CarePlan" />
+      <short value="HelpFromOthers" />
+      <alias value="HulpVanAnderen" />
+      <mapping>
+        <identity value="zib-helpfromothers-v3.01-2020EN" />
+        <map value="NL-CM:3.2.1" />
+        <comment value="HelpFromOthers" />
+      </mapping>
+    </element>
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-HelpFromOthers.xml:41]: Fix (thanks for writing the text)

File: resources/zib/zib-HelpFromOthers.xml [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Practitioner" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Organization" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/CareTeam" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/HealthcareService" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-Practitioner" />
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-HelpFromOthers.xml:112]: Fix

File: resources/zib/zib-HelpFromOthers.xml (hunk 1 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-HelpFromOthers.xml:114]: Fix

File: resources/zib/zib-HelpFromOthers.xml [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+      <mapping>
+        <identity value="zib-helpfromothers-v3.01-2020EN" />
+        <map value="NL-CM:3.2.2" />
+        <comment value="Frequency" />
+      </mapping>
+    </element>
+    <element id="CarePlan.activity.detail.performer">
+      <path value="CarePlan.activity.detail.performer" />
+      <short value="CareAid" />
+      <definition value="An CareAid can be a health professional such as nurses, carers and helpers but also a caregiver or healthcare provider such as a home care organization. Either a reference to details of the health professional involved in the patient’s care, the details of the caregiver involved in the patient’s support or the details of the organization the health professional/careaid works for in the event that it is not entered as a person." />
```
[the remark this answers is not in the document]
Comment (Ashwin-nictiz) [reply resources/zib/zib-HelpFromOthers.xml:96]: Fix

File: resources/zib/zib-HelpFromOthers.xml (hunk 1 above)
Comment (Ashwin-nictiz) [reply resources/zib/zib-HelpFromOthers.xml:114]: This is not possible because location is not supported in the target profiles

**Review by LuudSlagter — CHANGES_REQUESTED** [review]

**Comment by LuudSlagter** [comment]
Should we add some guidance on `.activity.detail.status`? In the ada2FHIR I've fixed this element to _in-progress_, since that seems best in line with `.status` equal to _active_. If we decide to add guidance, we should add it to zib-NursingIntervention as well.

File: examples/nl-core-HelpFromOthers-01.xml [hunk 8]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CarePlan xmlns="http://hl7.org/fhir">
```
Comment (Ashwin-nictiz) [inline examples/nl-core-HelpFromOthers-01.xml:2]: The tool gives the following error:

```
  -  error at CarePlan.activity[0].detail.performer[0] (32, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-ContactPerson')): The URL 'RelatedPerson/nl-core-HelpFromOthers-01-RelatedPerson-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.activity[0].detail.performer[0] (32, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole')): The URL 'RelatedPerson/nl-core-HelpFromOthers-01-RelatedPerson-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.activity[0].detail.performer[0] (32, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthcareProvider-Organization')): The URL 'RelatedPerson/nl-core-HelpFromOthers-01-RelatedPerson-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
```

File: examples/nl-core-HelpFromOthers-02.xml [hunk 9]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CarePlan xmlns="http://hl7.org/fhir">
```
Comment (Ashwin-nictiz) [inline examples/nl-core-HelpFromOthers-02.xml:2]: The tools gives the following error:

```
  -  error at CarePlan.activity[0].detail.performer[0] (32, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-ContactPerson')): The URL 'Organization/nl-core-HelpFromOthers-02-Organization-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.activity[0].detail.performer[0] (32, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole')): The URL 'Organization/nl-core-HelpFromOthers-02-Organization-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.activity[0].detail.performer[0] (32, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-HelpFromOthers path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthcareProvider-Organization')): The URL 'Organization/nl-core-HelpFromOthers-02-Organization-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
```

File: examples/nl-core-HelpFromOthers-01.xml (hunk 8 above)
Comment (LuudSlagter) [reply examples/nl-core-HelpFromOthers-01.xml:2]: These errors are not present anymore.

File: examples/nl-core-HelpFromOthers-02.xml (hunk 9 above)
Comment (LuudSlagter) [reply examples/nl-core-HelpFromOthers-02.xml:2]: These errors are not present anymore.

**Pull request merged**